packages/main/src/Menu.ts

```diff
@@ -255,26 +258,86 @@ class Menu extends UI5Element {
 		return isPhone();
 	}
 
+	get acessibleNameText() {
+		return Menu.i18nBundle.getText(MENU_POPOVER_ACCESSIBLE_NAME);
+	}
+
 	get _popover() {
 		return this.shadowRoot!.querySelector<ResponsivePopover>("[ui5-responsive-popover]")!;
 	}
+	get _list() {
+		return this.shadowRoot!.querySelector<List>("[ui5-list]");
+	}
+
+	/** Returns menu item groups */
+	get _menuItemGroups() {
+		return this.items.filter((item) : item is MenuItemGroup => item.isGroup);
```

Comment:
it is good to extract typeguard in separate function outside of the class which could be used between different files (if needed)

Reply:
Done